tools/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ else()
   ignore_llvm_tool_subdirectory(llvm-shlib)
 endif()
 
-# add_llvm_tool_subdirectory(opt) # HLSL Change
+add_llvm_tool_subdirectory(opt) # HLSL Change
```

Comment:
It seems opt depends on DxilConvPasses, which depends on Windows stuff. So this will need to be condition on WIN32 for Linux's sake.

Reply:
I know, I just did have time to push it yet :)